docs/meshstack.aws.metering.md

```diff
@@ -30,3 +32,223 @@ In other words, `Credit`, `Refund`, `RIFee`, `Tax` `SavingsPlanUpfrontFee`, `Sav
 `Anniversary` are taken into the calculation. In other words, line items with bill type `Purchase` and `Refund` are excluded.
 * For each line item, we take the [effective cost](https://docs.aws.amazon.com/cur/latest/userguide/reservation-columns.html#r-E)
 when available and [unblended cost](https://docs.aws.amazon.com/cur/latest/userguide/Lineitem-columns.html#l-U) otherwise
+
+## IAM User Configuration
+
+### Policies
+
+In order for meshStack to process AWS Cost and Usage Reports, following policies are required:
+
+1. **Bucket Access Policy**: This policy allows the Metering IAM user to fetch AWS cost and usage reports
+2. **Organization Access Policy**: This policy allows the Metering IAM user to list all accounts in the organization
+3. **Assume Role Policy**: This policy allows Metering IAM user to assume the IAM Role `MeteringRole`
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Bucket Access Policy-->
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
```

Comment:
q: I suppose this was taken from the metering role in our dev env? We probably don't need all these permissions. Is it a predefined policy on AWS?

Reply:
Yes it was taken from the dev env, but the permissions are the same ones we use in our [terraform platform module](https://github.com/meshcloud/meshfed-release/blob/develop/deployment/deployments/tf-platform-modules/aws/meshcloud-cost-usage-reports-iam-user/iam.tf#L19-L53) in which we create a custom AWS policy

Reply:
As discussed in our call, the unnecessary permissions are removed, thanks!